docker/serverAndUI/Dockerfile

```diff
@@ -51,4 +51,4 @@ RUN git clone https://github.com/Netflix/conductor.git \
 EXPOSE 5000 8080
 
 CMD ["/app/startup.sh"]
-ENTRYPOINT ["/bin/bash"]
+ENTRYPOINT ["/bin/bash"]
```

Comment:
super nitpick: there should be a newline at the end of this line (hence github adding the symbol at the end)